docs/workflow_tasks/files_to_granules.md

```diff
@@ -0,0 +1,50 @@
+---
+id: files_to_granules
+title: Files To Granules
+hide_title: true
+---
+
+# Files To Granules
+
+This task utilizes the Cumulus Message Adapter to interpret and construct incoming and outgoing messages.
+
+Links to the npm package, task input, output and configuration schema definitions and more can be found on the auto-generated [Cumulus Tasks](../tasks) page.
+
+## Summary
+
+This task utilizes the incoming ```config.input_granules``` and the task input list of s3 URIs along with the rest of the configuration objects to take the list of incoming files and sort them into a list of granule objects.
```

Comment:
`config.inputGranules` (_typically_ sourced from `meta.input_granules`, ugh) Sorry, I was wondering whether this might cause confusion but from prior PR feedback it seemed preferable to converge on camelCase for config variable names.

Reply:
a2e05a990